superset/explore/commands/get.py

```diff
@@ -152,11 +153,6 @@ def run(self) -> Optional[Dict[str, Any]]:
         except (SupersetException, SQLAlchemyError):
             dataset_data = dummy_dataset_data
 
-        if dataset:
```

Comment:
@hughhhh `DatasourceEditor` uses it, without it you would get the `Owners is invalid` error every time you try to edit a datasource in explore, so before putting back I'd like to ask if there was any particular reason why we removed it?